images/node/Dockerfile

```diff
@@ -17,6 +17,7 @@ ADD https://copr.fedoraproject.org/coprs/maxamillion/origin-next/repo/epel-7/max
 RUN yum install -y libmnl libnetfilter_conntrack openvswitch \
     libnfnetlink iptables iproute bridge-utils procps-ng ethtool socat openssl \
     binutils xz kmod-libs kmod sysvinit-tools device-mapper-libs dbus \
+    e2fsprogs xfsprogs ceph-common glusterfs-fuse iscsi-initiator-utils nfs-utils \
```

Comment:
I really don't like bloating this with `ceph-common glusterfs-fuse iscsi-initiator-utils nfs-utils`. I'd rather see QA show us that they are needed rather than add them universally and have to deal with the maintainance of images with these.

Due to the current design of the containerized kubelet we know that we will need e2fsprogs and xfsprogs. I think that's an incorrect design, but doing this should be a safe way to get us past it.